app/components/admin/project/admin-project.controller.js

```diff
@@ -18,6 +18,10 @@
     showProjects();
 
     function showProjects() {
+      if (UserService.islogged()) {
+        ngNotify.set('session required', 'error');
+        $state.go('login');
+      }
```

Comment:
Move this to the `app.run` so every time the app loads a new page it will validate if the session is active.